example.csharp/addons/tiltfive/scenes/T5ControllerCS.cs

```diff
@@ -0,0 +1,25 @@
+using Godot;
+using System;
+
+public partial class T5ControllerCS : Node3D
```

Comment:
This might be good to mark as a [Tool] otherwise it's not possible to implement editor checks to verify nodes are correctly added under controllers.